.env

```diff
@@ -0,0 +1,2 @@
+REGISTRY=quay.io/nvim-lsp/
```

Comment:
Hmm why would this be here instead of embedded into the compose file?

Reply:
Easier maintenance. Less copy paste.
Compose automatically picks up vars in .env and uses them.
If we ever want to change to a different repo or registry, we just have to update the env file, instead of each and every line in the compose.